cmd/pctl/main.go

```diff
@@ -122,6 +124,28 @@ func installCmd() *cli.Command {
 				DefaultText: "profile_subscription.yaml",
 				Usage:       "Filename to use for the generated content.",
 			},
+			&cli.BoolFlag{
+				Name:  "create-pr",
+				Value: false,
+				Usage: "If given, install will create a PR for the modifications it outputs.",
+			},
+			&cli.StringFlag{
+				Name:        "remote",
+				Value:       "origin",
+				DefaultText: "origin",
+				Usage:       "The remote to push the branch to.",
+			},
+			&cli.StringFlag{
+				Name:        "base",
+				Value:       "main",
+				DefaultText: "main",
+				Usage:       "The base branch to open a PR against.",
+			},
+			&cli.StringFlag{
+				Name:  "repo",
+				Value: "",
+				Usage: "The repository to open a pr against. Format is: org/repo-name",
+			},
```

Comment:
can't this be discovered from the remote?

Reply:
It can but if it is a fork or there are multiple remotes, which one do you choose?

Reply:
ah gotcha :+1: